### PR TITLE
18 Put Volunteer

### LINKED ISSUE #18: Add volunteer to request
PUT /requests/{requestId}/volunteers/{userId}

### PR DESCRIPTION
**Pull request by H4kor**
Lambda ist angelegt.

### CODE REVIEW

File: common/help-on-spot-models/src/ormConfig.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
   "type": "postgres",
   "host": process.env.DATABASE_HOST,
-  "port": 5432,
-  "username": process.env.DATABASE_USERNAME,
-  "password": process.env.DATABASE_PASSWORD,
-  "database": process.env.DATABASE_NAME
+  "port": 6000,
+  "username": "postgres",
+  "password": "postgres",
+  "database": "postgres"
```
Comment (meteohr) [inline common/help-on-spot-models/src/ormConfig.ts]: Please don't change the ormConfig, create a `.env` file instead.

File: lambdas/put-volunteer/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    const savedResponse = await connection!.getRepository(RequestResponse).save(rResp);
+    return lambdaResponse(200, {
+        id: savedResponse.id,
+        status: savedResponse.status,
+        createTime: savedResponse.createTime,
+        updateTime: savedResponse.updateTime,
+        userId: savedResponse.userId,
+        request: request,
+    });
+}
```
Comment (izi01) [inline lambdas/put-volunteer/src/index.ts:73]: connection should be closed at the end of the handler

File: common/help-on-spot-models/src/ormConfig.ts (hunk 1 above)
Comment (H4kor) [reply common/help-on-spot-models/src/ormConfig.ts]: Oops. Shouldn't have have been commited.

File: lambdas/put-volunteer/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "post-volunteer",
```
Comment (izi01) [inline lambdas/put-volunteer/package.json]: should be 'put-volunteer'

**Review by izi01 — APPROVED** [review]

**Pull request merged**